doc/api/v0.yaml

```diff
@@ -58,6 +58,29 @@ paths:
           description: The requested fund was not found
 
   /api/v0/proposals:
+    post:
+      summary: Get proposal by id
+      tags: [ proposal ]
+      description: |
+        Retrieves queried proposals.
+      requestBody:
+        description: List of voteplan id and indexes query
+        required: true
+        content:
+          application/json:
+            schema:
+                $ref: '#/components/schemas/ProposalsByVoteplanIdAndIndexQuery'
+
+      responses:
+        "200":
+          description: Valid response
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProposalWithChallengeInfo"
+        "404":
+          description: The requested proposals were not found
```

Comment:
all of them? or any of them?

Reply:
As per now it is any. Will update.

Reply:
Actually this was missplaced. If something is not found it will just not appear. Query may return an empty list.